lib/data/authentication_service.dart

```diff
@@ -58,31 +59,32 @@ class AuthenticationService {
     // Create a GitHubSignIn instance
     final GitHubSignIn gitHubSignIn = GitHubSignIn(
         // Input the staging info from the Teams app channel
-        clientId: '',
-        clientSecret: '',
-        redirectUrl: '');
+        clientId: Config.gitHubClientId,
+        clientSecret: Config.gitHubClientSecret,
+        redirectUrl: Config.gitHubCallbackUrl);
 
     // Trigger the sign-in flow
-    final result = await gitHubSignIn.signIn(context);
+    var result = await gitHubSignIn.signIn(context);
 
     // Create a credential from the access token
-    final githubAuthCredential = GithubAuthProvider.credential(result.token);
 
-    /*
     switch (result.status) {
       case GitHubSignInResultStatus.ok:
         print(result.token);
-        break;
+        final githubAuthCredential =
+            GithubAuthProvider.credential(result.token);
+            
+        // Once signed in, return the UserCredential
+        return await FirebaseAuth.instance
+            .signInWithCredential(githubAuthCredential);
 
       case GitHubSignInResultStatus.cancelled:
       case GitHubSignInResultStatus.failed:
         print(result.errorMessage);
         break;
     }
-    */
 
-    // Once signed in, return the UserCredential
-    return await FirebaseAuth.instance
-        .signInWithCredential(githubAuthCredential);
```

Comment:
App was crashing if sign in process was cancelled with this being returned for all cases